workspaces/tekton/plugins/tekton/src/components/pipeline-topology/PipelineVisualizationStepList.tsx

```diff
@@ -59,7 +62,7 @@ export const PipelineVisualizationStepList = ({
       )}
       {isFinallyTask && (
         <div className="bs-tkn-pipeline-visualization-step-list__task-type">
-          Finally task
+          {t('pipelineVisualization.setpList.finallyTaskTitle')}
```

Comment:
Was this meant to be `pipelineVisualization.stepList.finallyTaskTitle` instead of `pipelineVisualization.setpList.finallyTaskTitle` ?

Reply:
yes, fixed it